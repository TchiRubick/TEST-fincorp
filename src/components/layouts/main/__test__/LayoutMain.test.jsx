import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import LayoutMain from '../LayoutMain';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<LayoutMain>test</LayoutMain>, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<LayoutMain>test</LayoutMain>).toJSON();
  expect(tree).toMatchSnapshot();
});
