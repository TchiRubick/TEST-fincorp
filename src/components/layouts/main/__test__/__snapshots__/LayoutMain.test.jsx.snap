// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="layout-main"
>
  test
</div>
`;
